spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// Contributors:
+// Emily Jiang
+
+= Release Notes
+
+[[release_notes_2]]
+== Release Notes for MicroProfile Health Check 2.0
+
+The following changes occurred in the 2.0 release, compared to 1.0
+
+A full list of changes may be found on the link:https://github.com/eclipse/microprofile-health/issues?utf8=✓&q=is%3Aissue+milestone%3A2.0+[MicroProfile Health Check 2.0]
```

Comment:
maybe it would be better to list here all the commits between releases? We don't have issue for every change.  
Something like https://github.com/eclipse/microprofile-health/compare/1.0...2.0-RC1 maybe? (of course replacing 2.0-RC1 with GA tag). Or maybe this is requirement from release doc instructions?

Reply:
Even if we don't have an issue for each commit, the most important features are in issues. Some commit may be more confusing than useful (reverting stuff for example)